src/clj/ladders/elo.clj

```diff
@@ -0,0 +1,63 @@
+(ns ladders.elo)
+
+;; q-term name taken from formula on wikipedia page for elo-rating
```

Comment:
Here's a link to how to write docstrings in a defn from the Clojure Style Guide.

https://github.com/bbatsov/clojure-style-guide#comments